component/conprof/http/api.go

```diff
@@ -124,9 +124,10 @@ func getProfileEstimateSize(component topology.Component) int {
 }
 
 var (
-	QueryStateSuccess       string = "success"
-	QueryStateFailed        string = "failed"
-	QueryStatePartialFailed string = "partial failed"
+	QueryStateSuccess        string = "success"
+	QueryStatePartialSuccess string = "partial success"
+	QueryStateFailed         string = "failed"
+	QueryStatePartialFailed  string = "partial failed"
```

Comment:
For constant enums maybe you should use some tokens without any spaces, e.g. `partial_failed`, instead of something looks like a human readable description. Otherwise people may want to incorrectly make it more readable some day, e.g. changing it to be "partial failed because xxxxx". Don't expose unnecessary risk of making mistakes.